app/uk/gov/hmrc/taxhistory/services/EmploymentHistoryService.scala

```diff
@@ -181,6 +222,10 @@ class EmploymentHistoryService @Inject()(
   def getNpsIabds(nino: Nino, taxYear: TaxYear)(implicit hc: HeaderCarrier): Future[List[Iabd]] =
     npsConnector.getIabds(nino, taxYear.currentYear)
 
+  /**
+    * If the tax year requested is not the previous tax year, return None.
+    * TODO is this the correct behaviour?
```

Comment:
We only display the Tax Account information for CY-1, which is why I believe this is here.